atom/browser/ui/atom_menu_model.h

```diff
@@ -35,10 +48,16 @@ class AtomMenuModel : public ui::SimpleMenuModel {
 
   void SetRole(int index, const base::string16& role);
   base::string16 GetRoleAt(int index);
+  bool GetAcceleratorAtWithParams(int index,
+                                  bool use_default_accelerator,
+                                  ui::Accelerator* accelerator) const;
 
   // ui::SimpleMenuModel:
   void MenuClosed() override;
 
+  using SimpleMenuModel::GetSubmenuModelAt;
+  AtomMenuModel* GetSubmenuModelAt(int index);
```

Comment:
@zcbenz I overloaded this and narrowed it to return an `AtomMenuModel` instead of a `ui::MenuModel` so calling methods didn't need to call `static_cast` themselves.

Let me know if you are okay with this or would prefer the calling method cast the return value down.